examples/stable_diffusion_xl/demo/sampling_without_streamlit.py

```diff
@@ -36,6 +38,14 @@ def get_parser_sample():
     parser.add_argument("--negative_aesthetic_score", type=float, default=None)
     parser.add_argument("--sampler", type=str, default="EulerEDMSampler")
     parser.add_argument("--guider", type=str, default="VanillaCFG")
+    parser.add_argument(
+        "--guidance_scale",
+        type=float,
+        default=5.0,
+        help="unconditional guidance scale: "
+        "eps = eps(x, uncond) + scale * (eps(x, cond) - eps(x, uncond)). "
+        "Simplified: `uc + scale * (uc - prompt)`",
```

Comment:
`uc + scale * (uc - prompt)` -> For NoDynamicThresholding, `uncond + guidance_scale * (uncond - cond)`

Reply:
updated